docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -286,254 +286,568 @@ visitor_rate_ecs::
 [[ootb-ml-jobs-siem]]
 === SIEM
 
-These {anomaly-jobs} appear by default in the Anomaly Detection interface of the 
-{siem-guide}/machine-learning.html[SIEM app] in {kib}. They help you 
-automatically detect file system and network anomalies on your hosts. The list
-below contains the jobs organized by `agent.type` ({auditbeat}, {packetbeat},
-and {winlogbeat}).
+These {anomaly-jobs} appear by default in the Anomaly Detection interface of
+the {siem-guide}/machine-learning.html[SIEM app] in {kib}. They help you 
+automatically detect file system and network anomalies on your hosts.
```

Comment:
@szabosteve @lcawl 
Don't know if you want to add some info here as to why the ECS fields are listed for each job?

Reply:
@benskelker @lcawl 
Sure, I'll add in a separate PR that when you are not using Beats, you need to map your data to the respective ECS fields.

Reply:
@benskelker @lcawl 
I added a sentence about the listed ECS fields as a single comment to avoid conflicts. Please let me know if you would modify or rather omit it.

Reply:
Thanks @szabosteve